distribution/docs/src/main/resources/_contents/_transformers/tika-input-transformer-contents.adoc

```diff
@@ -0,0 +1,61 @@
+==== Tika Input Transformer
+
+The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into a Catalog Metacard.
```

Comment:
Yes. I've also made an appendix of all the formats available with the Tika transformer, but it may not make the final edit.